test/unit/default_queue_tests.rb

```diff
@@ -69,18 +91,10 @@ class UnitTests < Assert::Context
       assert subject.empty?
     end
 
-    should "reset its shutdown flag when started" do
-      assert_false subject.shutdown?
-      subject.shutdown
-      assert_true subject.shutdown?
-      subject.start
-      assert_false subject.shutdown?
-    end
-
   end
 
-  class SignallingTests < UnitTests
-    desc "mutex and condition variable behavior"
+  class SignallingTests < StartedTests
+    desc "its mutex and condition variable behavior"
```

Comment:
@jcredding I'd just remove the description here - the test descriptions cover things well enough in this case.